### PR TITLE
feat(nx-dev): powerpack landing page

### PR DESCRIPTION
**Pull request by juristr**
## Related Issue(s)

Fixes #

### CODE REVIEW

File: nx-dev/nx-dev/app/powerpack/page.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  Hero,
+  PowerpackFeatures,
+} from '@nx/nx-dev/ui-powerpack';
+
+import type { Metadata } from 'next';
+
+export const metadata: Metadata = {
+  title: 'Nx PowerPack',
+  description:
+    'Nx PowerPack is a suite of advanced tools designed for enterprise use cases.',
```
Comment (isaacplmann) [inline nx-dev/nx-dev/app/powerpack/page.tsx:14]:
```suggestion
    'Nx PowerPack is a suite of paid extensions for the Nx CLI specifically designed for enterprises.',
```

File: nx-dev/nx-dev/app/powerpack/page.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+  openGraph: {
+    url: 'https://nx.dev/powerpack',
+    title: 'Nx PowerPack',
+    description:
+      'Nx PowerPack is a suite of advanced tools designed for enterprise use cases.',
```
Comment (isaacplmann) [inline nx-dev/nx-dev/app/powerpack/page.tsx:19]:
```suggestion
      'Nx PowerPack is a suite of paid extensions for the Nx CLI specifically designed for enterprises.',
```

File: nx-dev/ui-common/src/lib/headers/menu-items.ts [hunk 3]
```diff
@@ -113,6 +113,15 @@ export const plans: MenuItem[] = [
     isNew: false,
     isHighlight: false,
   },
+  {
+    name: 'Nx Powerpack',
+    description:
+      'A suite of paid extensions for Nx specifically designed for enterprises.',
```
Comment (isaacplmann) [inline nx-dev/ui-common/src/lib/headers/menu-items.ts:119]:
```suggestion
      'A suite of paid extensions for the Nx CLI specifically designed for enterprises.',
```

File: nx-dev/ui-powerpack/src/lib/get-started.tsx [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+                    title="Self-hosted cache storage"
+                  >
+                    Self-hosted cache storage
+                  </TextLink>
+                  ,{' '}
+                  <TextLink
+                    href="#workspace-conformance"
+                    title="Workspace conformance"
+                  >
+                    wokrspace conformance
```
Comment (isaacplmann) [inline nx-dev/ui-powerpack/src/lib/get-started.tsx:154]:
```suggestion
                    workspace conformance
```

File: nx-dev/ui-powerpack/src/lib/hero.tsx [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+export function Hero(): ReactElement {
+  return (
+    <div className="mx-auto max-w-7xl px-6 lg:px-8">
+      <div className="mx-auto max-w-2xl text-center">
+        <SectionHeading as="h1" variant="display">
+          Nx Powerpack
+        </SectionHeading>
+        <SectionHeading as="p" variant="subtitle" className="mt-6 text-center">
+          A suite of paid extensions for Nx specifically designed for
```
Comment (isaacplmann) [inline nx-dev/ui-powerpack/src/lib/hero.tsx:13]:
```suggestion
          A suite of paid extensions for the Nx CLI specifically designed for
```

File: nx-dev/ui-powerpack/src/lib/powerpack-features.tsx [hunk 6]
```diff
@@ -0,0 +1,262 @@
...
+              <SectionHeading
+                as="h2"
+                variant="title"
+                id="self-hosted-cache-storage"
+                className="scroll-mt-48"
+              >
+                Self-hosted cache storage
+              </SectionHeading>
+              <p className="mt-6 text-pretty text-lg">
+                Nx Powerpack enables you to use <Strong>AWS S3</Strong> or a{' '}
```
Comment (isaacplmann) [inline nx-dev/ui-powerpack/src/lib/powerpack-features.tsx:29]:
```suggestion
                Nx Powerpack enables you to use <Strong>Amazon S3</Strong> or a{' '}
```

File: nx-dev/ui-powerpack/src/lib/powerpack-features.tsx [hunk 7]
```diff
@@ -0,0 +1,262 @@
...
+                <Strong>custom network drive</Strong> as your remote cache
```
Comment (isaacplmann) [inline nx-dev/ui-powerpack/src/lib/powerpack-features.tsx:30]:
```suggestion
                <Strong>shared network drive</Strong> as your remote cache
```

**Review by isaacplmann — COMMENTED** [review]
- I think Powerpack should be described as extensions to `the Nx CLI` and not just `Nx`.  That makes it clearer what the distinction is between Powerpack and Nx Cloud.
- Amazon refers to their S3 product as Amazon S3 not AWS S3
- A few typo fixes

File: nx-dev/ui-powerpack/src/lib/powerpack-features.tsx (hunk 6 above)
Comment (juristr) [reply nx-dev/ui-powerpack/src/lib/powerpack-features.tsx:29]: doesn't probably matter a lot, but in the docs we also mention mostly "AWS S3" everywhere

**Review by FrozenPandaz — APPROVED** [review]

**Pull request merged**